### PR TITLE
Fix docstrings for NLU class

### LINKED ISSUE #28: Fix docstrings for NLU class

### PR DESCRIPTION
**Pull request by NoB0**
Fixes #28

### CODE REVIEW

File: dialoguekit/nlu/models/diet_classifier_rasa.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 More information about the DIET classifier
 https://rasa.com/docs/rasa/reference/rasa/nlu/classifiers/diet_classifier/
 
-A short description of how we use Rasa as a component library can be seen inn
+A short description of how we use Rasa as a component library can be seen in
 the docs/rasa_component_library.md
```
Comment (kbalog) [inline dialoguekit/nlu/models/diet_classifier_rasa.py:7]: Nit: drop "the"
That file is no longer there, can you link https://iai-group.github.io/DialogueKit/nlu.html#rasa-as-a-component-library instead?

File: dialoguekit/nlu/models/satisfaction_classifier.py [hunk 2]
```diff
@@ -25,7 +26,7 @@ def _tokenize_predict(self, input_text: List[str]) -> List[int]:
         in each string by it self.
 
         Args:
-            input_text: list of text to classified
+            input_text: list of text to classified.
```
Comment (kbalog) [inline dialoguekit/nlu/models/satisfaction_classifier.py:29]: Nit: capitalize List

File: dialoguekit/nlu/models/satisfaction_classifier.py [hunk 3]
```diff
@@ -125,11 +126,12 @@ def classify_text(
     def classify_last_n_dialogue(
         self, dialogue: Dialogue, last_n: Optional[Union[int, None]] = None
     ) -> int:
-        """Classify n last dialogues.
+        """Classifies n last dialogues.
```
Comment (kbalog) [inline dialoguekit/nlu/models/satisfaction_classifier.py:129]: Nit: "Classifies the last dialogues."

**Review by kbalog — APPROVED** [review]
LGTM

File: dialoguekit/nlu/models/diet_classifier_rasa.py (hunk 1 above)
Comment (NoB0) [reply dialoguekit/nlu/models/diet_classifier_rasa.py]: Done.

File: dialoguekit/nlu/models/satisfaction_classifier.py (hunk 2 above)
Comment (NoB0) [reply dialoguekit/nlu/models/satisfaction_classifier.py]: Done.

File: dialoguekit/nlu/models/satisfaction_classifier.py (hunk 3 above)
Comment (NoB0) [reply dialoguekit/nlu/models/satisfaction_classifier.py]: Done.

**Pull request merged**